src/commands/appSettings/downloadAppSettings.ts

```diff
@@ -22,27 +24,31 @@ export async function downloadAppSettings(node?: AppSettingsTreeItem): Promise<v
     const client: SiteClient = node.root.client;
 
     const message: string = 'Select the destination file for your downloaded settings.';
-    const envVarPath: string = await workspaceUtil.selectWorkspaceFile(ext.ui, message, () => envFileName);
+    const envVarPath: string = await workspaceUtil.selectWorkspaceFile(message, () => envFileName);
     const envVarUri: vscode.Uri = vscode.Uri.file(envVarPath);
 
     await node.runWithTemporaryDescription('Downloading...', async () => {
         ext.outputChannel.appendLine(`Downloading settings from "${client.fullName}"...`);
         const localEnvVariables: DotenvParseOutput = await getLocalEnvironmentVariables(envVarPath, true /* allowOverwrite */);
         const remoteEnvVariables: WebSiteManagementModels.StringDictionary = await client.listApplicationSettings();
         if (remoteEnvVariables.properties) {
-            await node.confirmOverwriteSettings(remoteEnvVariables.properties, localEnvVariables, envFileName);
+            await confirmOverwriteSettings(remoteEnvVariables.properties, localEnvVariables, envFileName);
         }
 
         await fse.ensureFile(envVarPath);
         await fse.writeFile(envVarPath, convertAppSettingsToEnvVariables(localEnvVariables, client.fullName));
     });
 
+    const input: string | undefined = await window.showInformationMessage(`Downloaded settings from "${client.fullName}".  View settings file?`, 'Open file');
```

Comment:
No strong preference between the two, but this should be consistent between "Open" and "View"